docker-compose/Makefile

```diff
@@ -167,6 +186,12 @@ init-whisk-cli:
 	echo "initializing CLI ... "
 	$(WSK_CLI) -v property set --namespace guest --auth `cat $(PROJECT_HOME)/ansible/files/auth.guest` --apihost $(DOCKER_HOST_IP):443 -i
 
+
+.PHONY: init-catalog
+init-catalog:
+	cd $(CATALOG_HOME) && \
```

Comment:
I got this to work by using the following `init-catalog` script:

```
init-catalog:
	OPENWHISK_HOME=$(PROJECT_HOME) $(CATALOG_HOME)/packages/installCatalog.sh \
	  `cat $(PROJECT_HOME)/ansible/files/auth.whisk.system` \
	  $(DOCKER_HOST_IP):443 \
	  $(WSK_CLI)
```
thanks @Shillaker for the inspiration 😄